platform-modules/mimetype-manager/src/main/scala/org/sunbird/mimetype/mgr/impl/AssetMimeTypeMgrImpl.scala

```diff
@@ -12,22 +12,22 @@ import scala.concurrent.{ExecutionContext, Future}
 
 class AssetMimeTypeMgrImpl(implicit ss: StorageService) extends BaseMimeTypeManager with MimeTypeManager {
 
-	override def upload(objectId: String, node: Node, uploadFile: File)(implicit ec: ExecutionContext): Future[Map[String, AnyRef]] = {
+	override def upload(objectId: String, node: Node, uploadFile: File, filePath: Option[String])(implicit ec: ExecutionContext): Future[Map[String, AnyRef]] = {
 		validateUploadRequest(objectId, node, uploadFile)
 		val fileMimeType = getFileMimeType(uploadFile)
 		val nodeMimeType = node.getMetadata.getOrDefault("mimeType", "").asInstanceOf[String]
 		TelemetryManager.log("Uploading Asset MimeType: " + fileMimeType)
 		//TODO: Throw Client Exception Here
 		if (!StringUtils.equalsIgnoreCase(fileMimeType, nodeMimeType))
 			TelemetryManager.log("Uploaded File MimeType is not same as Node (Object) MimeType. [Uploading MimeType: " + fileMimeType + " | Node (Object) MimeType: " + nodeMimeType + "]")
-		val result: Array[String] = uploadArtifactToCloud(uploadFile, objectId)
+		val result: Array[String] = uploadArtifactToCloud(uploadFile, objectId, filePath)
```

Comment:
Here in result, we have to add one new field `artifactBasePath`, till `/content`
But only when filePath is not empty or blank

`"artifactBasePath": "https://sunbirddev.blob.core.windows.net/sunbird-content-dev/xyz/content"`

Similar to AssetMimeTypeMgrImpl, this has to be done for other manager.